tests/unit/serializers/ember-data-serializer-discover-test.js

```diff
@@ -1,6 +1,7 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
-import {createServer, applyEmberDataSerializers} from 'ember-cli-mirage';
+import {createServer } from 'miragejs';
```

Comment:
@cah-briangantzler minor, missing space in `{createServer }`

Reply:
Fixed